src/target/merge-package.ts

```diff
@@ -24,13 +23,13 @@ export const mergePackageJson = (
         target.devDependencies = {};
     }
 
-    addNewOrUnchanged(target.dependencies, newContent.dependencies, lastContent?.dependencies ?? null);
-    addNewOrUnchanged(target.devDependencies, newContent.devDependencies, lastContent?.devDependencies ?? null);
+    addNewOrUnchanged(target.dependencies, newContent.dependencies || {}, lastContent?.dependencies ?? null);
+    addNewOrUnchanged(target.devDependencies, newContent.devDependencies || {}, lastContent?.devDependencies ?? null);
```

Comment:
Fix to avoid enumerating on undefined:

```
> Object.entries(undefined)
Uncaught TypeError: Cannot convert undefined or null to object
```

Reply:
Could `target` be missing any of `dependencies`, `devDependencies` or `scripts`? Should the `|| {}` be used also on the first argument in these calls?

Reply:
The lines just above make (L19, L23) sure those properties are initialized so it can be changed in these calls. Same for scripts on L30.